File: python/examples/03_core_id.py
from uiid_client import UIIDClient

def main():
    print("=== 03. Core ID API Workflow (Python) ===\n")

    client = UIIDClient()
    print("[Core ID] Methods available in Python SDK:")
    print(" - client.generate_core_id() -> Generate public identity string")
    print(" - client.get_kyc_status() -> Fetch verification level & trust score")
    print(" - client.store_core_data(key='config', value='encrypted', is_public=False, is_immutable=True)")
    print(" - client.get_authorized_applications() -> Retrieve authorized third-party app list\n")

if __name__ == "__main__":
    main()

File: php/examples/03_core_id.php
<?php

require_once __DIR__ . '/../vendor/autoload.php';

use UIID\SDK\UIIDClient;

echo "=== 03. Core ID API Workflow (PHP) ===\n\n";

echo "[Core ID] Available PHP SDK Methods:\n";
echo " - \$client->generateCoreID()\n";
echo " - \$client->getKycStatus()\n";
echo " - \$client->storeCoreData('app_config', 'encrypted_data', isPublic: false, isImmutable: true)\n";
echo " - \$client->getAuthorizedApplications()\n\n";
